mopidy_listenbrainz/listenbrainz.py

```diff
@@ -232,15 +232,22 @@ def _collect_playlist_data(
             return None
         track_mbids = []
         for track_dto in dto.get("track", []):  # not tracks!
```

Comment:
should the default be a dict `{}`? we try to get named keys from it
```suggestion
        for track_dto in dto.get("track", {}):  # not tracks!
```

Reply:
I dont think this was changed in the PR so it's not a problem here but lmk if this should be changed.

Reply:
Good point!

Reply:
note for self: fixed in #11